### ISSUE TITLE
Keycloak 24.* introduced breaking changes for email-verification-by-code form

### ISSUE THREAD

**Issue by danieliu456** [RedFroggy/keycloak-verify-email-by-code#19]
After upgrading to keycloak 24.0.0 verify-email-by-code screen doesn't work.

```
<time> ERROR [org.keycloak.services.error.KeycloakErrorHandler] (executor-thread-31) Uncaught server error: java.lang.NoSuchMethodError: 'void org.keycloak.email.freemarker.beans.ProfileBean.<init>(org.keycloak.models.UserModel)'
at fr.redfroggy.keycloak.requiredactions.VerifyEmailByCode.createFormChallenge(VerifyEmailByCode.java:74)
	at fr.redfroggy.keycloak.requiredactions.VerifyEmailByCode.sendVerifyEmailAndCreateForm(VerifyEmailByCode.java:180)
```

according to docs constructor definition has changed

23.*
https://www.keycloak.org/docs-api/23.0.5/javadocs/org/keycloak/email/freemarker/beans/ProfileBean.html
![image](https://github.com/RedFroggy/keycloak-verify-email-by-code/assets/50437129/dcb9c70f-8f2f-4a79-b4a5-43b841b20324)

24.*
https://www.keycloak.org/docs-api/24.0.2/javadocs/org/keycloak/email/freemarker/beans/ProfileBean.html

![image](https://github.com/RedFroggy/keycloak-verify-email-by-code/assets/50437129/350929a1-d984-4780-97bd-936a621b537e)

Email sending works fine, but createFormChallenge fails to display form for email verification

**Comment by danieliu456** [comment]
Added fix: https://github.com/RedFroggy/keycloak-verify-email-by-code/pull/20

**Comment by dajay** [comment]
Hello,
New release is available [24.0.0](https://github.com/RedFroggy/keycloak-verify-email-by-code/releases/tag/24.0.0).
Thanks
